.github/ISSUE_TEMPLATE/05-other.md

```diff
@@ -0,0 +1,6 @@
+---
+name: '💭 Other'
+about: Issues that don't fit under anything else
+title: ''
+labels: ''
```

Comment:
Maybe we could add also these generic sections to guide the developer and to make it more structured for everyone:

```
What:

Why:

How:
```